src/InvestmentManager.sol

```diff
@@ -605,7 +605,7 @@ contract InvestmentManager is Auth {
             (uint8 currencyDecimals, uint8 trancheTokenDecimals) = _getPoolDecimals(liquidityPool);
 
             uint256 currencyAmountInPriceDecimals = _toPriceDecimals(trancheTokenAmount, trancheTokenDecimals).mulDiv(
-                price, 10 ** PRICE_DECIMALS, MathLib.Rounding.Down
+                price, 10 ** PRICE_DECIMALS, MathLib.Rounding.Up
```

Comment:
To be undone before merge: was added to verify that the invariant tests are working as expected.